### PR TITLE
Bump cats version.

### PR DESCRIPTION
**Pull request by saraiva132**
Bumps:

- cats
- cats-effect
- scalatest

### CODE REVIEW

File: core/src/main/scala/com/evolutiongaming/catshelper/DataHelper.scala [hunk 1]
```diff
@@ -9,7 +9,7 @@ object DataHelper {
 
   implicit class SortedMapOpsDataHelper[K, V](val self: SortedMap[K, V]) extends AnyVal {
 
-    def toNem(implicit order: Order[K]): Option[Nem[K, V]] = Nem.fromMap(self)
+    def toNem: Option[Nem[K, V]] = Nem.fromMap(self)
```
Comment (t3hnar) [inline core/src/main/scala/com/evolutiongaming/catshelper/DataHelper.scala:12]: it's not backward compatible change, you can add one more method and deprecate old one

File: core/src/test/scala/com/evolutiongaming/catshelper/ResourceBreakFlatMapChainTest.scala [hunk 2]
```diff
@@ -18,17 +18,17 @@ class ResourceBreakFlatMapChainTest extends AsyncFunSuite with Matchers {
     val result = ()
       .pure[IO]
       .toResource
-      .multiply(100)
```
Comment (t3hnar) [inline core/src/test/scala/com/evolutiongaming/catshelper/ResourceBreakFlatMapChainTest.scala:21]: please put 100 back to not have long running time

File: core/src/test/scala/com/evolutiongaming/catshelper/ResourceBreakFlatMapChainTest.scala [hunk 3]
```diff
@@ -18,17 +18,17 @@ class ResourceBreakFlatMapChainTest extends AsyncFunSuite with Matchers {
...
       .attempt
       .void
 
     FromFuture[IO]
       .apply { result.toFuture }
-      .timeout(1.second)
+      .timeout(3.second)
       .attempt
-      .flatMap { a => IO { a should matchPattern { case Left(_: TimeoutException) => } } }
+      .flatMap { a => IO { a should matchPattern { case Right(()) => } } }
```
Comment (t3hnar) [inline core/src/test/scala/com/evolutiongaming/catshelper/ResourceBreakFlatMapChainTest.scala:30]: @sirmax looks like https://github.com/typelevel/cats-effect/issues/469 is fixed now

**Review by t3hnar — CHANGES_REQUESTED** [review]

File: core/src/main/scala/com/evolutiongaming/catshelper/DataHelper.scala (hunk 1 above)
Comment (saraiva132) [reply core/src/main/scala/com/evolutiongaming/catshelper/DataHelper.scala]: Yeah. thanks

File: project/plugins.sbt [hunk 4]
```diff
@@ -8,4 +8,6 @@ addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.7")
 
 addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.13")
 
-addSbtPlugin("com.evolutiongaming" % "sbt-scalac-opts-plugin" % "0.0.5")
\ No newline at end of file
+addSbtPlugin("com.evolutiongaming" % "sbt-scalac-opts-plugin" % "0.0.5")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.5.1")
```
Comment (t3hnar) [inline project/plugins.sbt:13]: why do we need this here? @saraiva132 you could just add that in your local `.sbt/` folder

**Comment by saraiva132** [comment]
I think it's a super useful plugin. I can remove if you want, for this
project, might not make sense. Remove?

File: project/plugins.sbt (hunk 4 above)
Comment (saraiva132) [reply project/plugins.sbt]: I removed it ;)

Comment (t3hnar) [reply project/plugins.sbt]: it is indeed super useful! hence just add it locally to have for all projects, the idea is that you don't add optional stuff into project files, but rather into your local config :)

**Pull request merged**